recipes/cairosvg/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "CairoSVG" %}
+{% set version = "2.0.0rc6" %}
```

Comment:
Sure, done that. It is because the dependecies change quite a bit between the versions. I thought it is not worth investing time to package the old dependencies if the new version is released soon.